src/view/renderer.js

```diff
@@ -711,6 +711,8 @@ export default class Renderer {
 
 		if ( !container.parentElement || container.parentElement != domRoot ) {
 			domRoot.appendChild( container );
+		} else if ( !this._fakeSelectionNeedsUpdate( domRoot ) ) {
```

Comment:
This logic is a bit odd. First, we add the container and then we check whether it had to be updated? 

Also, inside `fakeSelecitonNeedsUpdate()` you check again whether `_fakeSelectionContainer` is defined... but we just did it here in `_updateFakeSelection()`

The logic should be reversed:

1. check whether we need to update anything; abort if not
2. if the container does not exist, create it
3. if the container isn't appended to the right root, append it